index.html

```diff
@@ -445,6 +445,382 @@ <h3 class="hosting__title">Wybierz swój plan hostingowy</h3>
 				</section>
 				<a class="hosting__text-last" href="">Warunki platnosci</a>
 			</section>
+			<section class="advantages">
+				<ul class="advantages__item">
```

Comment:
Lista z jendym elementem? To chyba nie lista. Dodatkwo nizej masz strong, namiast naglowka(h1-h6). To powinny byc cos takiego

<section>
 <div>
    <h4>Dział obsługi klienta 24/7/365</h4>
    <p>jakis tekst</p>
    <h4>Dział obsługi klienta 24/7/365</h4>
    <p>jakis tekst</p>
    <h4>Dział obsługi klienta 24/7/365</h4>
    <p>jakis tekst</p>
  <div>
 <div>
    <h4>Dział obsługi klienta 24/7/365</h4>
    <p>jakis tekst</p>
    <h4>Dział obsługi klienta 24/7/365</h4>
    <p>jakis tekst</p>
    <h4>Dział obsługi klienta 24/7/365</h4>
    <p>jakis tekst</p>
  <div>
</section>